src/main/webapp/chat.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>altruism-ator</title>
+    <link rel="stylesheet" href="chat-style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Karla&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <header>
+        <div class="container">
+            <nav>
+                <ul>
+                    <li class="nav-item"><a href="index.html">Home</a></li>
+                    <li class="nav-item"><a href="profile.html">Profile</a></li>
+                    <li class="nav-item"><a href="createYourOwn.html">Create your own</a></li>
+                    <li class="nav-item"><a href="chat.html">Chat</a></li>
+                </ul>
+            </nav>
+        </div>
+      </header>
+  </body>
+</html>
```

Comment:
If you could update the title of the page.

Reply:
Added the trailing line, however, I think the title should remain the same, since it is the name of the page overall and it is the one that appears on the little windows on the top bar.

Reply:
I will do something like:
altruism-ator - chat